nekoyume/Assets/_Scripts/Analyzer.cs

```diff
@@ -70,7 +69,10 @@ public class Analyzer
                 clientHash,
                 targetNetwork,
                 rpcServerHost);
+            FirebaseAnalytics.SetUserId(SystemInfo.deviceUniqueIdentifier);
```

Comment:
This line should move into the `SetAgentAddress()` method and follow Mixpanel's way.😉
```suggestion
```